acid/features/history/controller.py

```diff
@@ -14,10 +14,10 @@
 @builds.route('/builds/<int:page>')
 @db_session
 def show_builds_history(page=1):
-    per_page = current_app.config['buildset']['per_page']
+    per_page = current_app.config['history']['pagination']['per_page']
     pipeline = current_app.config['default']['pipename']
-    page_links = current_app.config['buildset']['page_links']
-    buildset_log_url = current_app.config['buildset']['log_url']
+    page_links = current_app.config['history']['pagination']['page_links']
+    buildset_log_url = current_app.config['history']['log_server']
```

Comment:
Consider changing the variable name to log_server_url. More consistent with config dictionary naming.